server/adaptors/integrations/__data__/repository/apache/info/README.md

```diff
@@ -0,0 +1,14 @@
+# Apache Integration
+
+## What is Apache ?
+Apache is an open source web server software for modern operating systems including UNIX and Windows.
+
+See additional details [here](https://httpd.apache.org/).
+
+## What is Apache Integration ?
+An integration is a bundle of pre-canned assets which are bundled togather in a meaningful manner.
+
+Apache integration includes dashboards, visualisations, queries and an index mapping.
```

Comment:
I gave this same comment to Lior earlier: in the READMEs we should have a description of how to set up an ingestion pipeline for this integration, if one is at all available. If one isn't available it's not a big deal.